dist/install-gleam.ps1

```diff
@@ -6,14 +6,16 @@ Set-Location $Env:RUNNER_TEMP
 
 $FILE_INPUT="gleam-${VSN}-windows-64bit.zip"
 $FILE_OUTPUT="gleam.zip"
-$DIR_FOR_BIN=".setup-beam/gleam/bin"
+$DIR_FOR_BIN=".setup-beam/gleam"
```

Comment:
The tweaking of `bin`, here, was basically for consistency with the other outputs (otherwise we'd have to attach the `bin` to Gleam's base folder).